services/QuillLMS/app/controllers/stripe_integration/webhooks_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module StripeIntegration
+  class WebhooksController < ApplicationController
+    protect_from_forgery except: :create
+
+    def create
+      Webhooks::HandleEventWorker.perform_async(stripe_webhook_event.id)
+      head 200
+    rescue ActiveRecord::RecordNotUnique
+      head 200
+    rescue => e
+      NewRelic::Agent.notice_error(e)
```

Comment:
A few days ago, I added a `ErrorNotifier.report(error)`. I'd like to start using that in other places (we log them in both Sentry and NR for now).
```suggestion
      ErrorNotifier.report(e)
```

Reply:
Cool!  I will update.